lib/git_reflow/git_helpers.rb

```diff
@@ -40,11 +40,24 @@ def fetch_destination(destination_branch)
       run_command_with_label "git fetch origin #{destination_branch}"
     end
 
-    def update_destination(destination_branch)
-      origin_branch = current_branch
-      run_command_with_label "git checkout #{destination_branch}"
-      run_command_with_label "git pull origin #{destination_branch}"
-      run_command_with_label "git checkout #{origin_branch}"
+    def update_destination(options = {})
```

Comment:
Let's leave the `destination_branch` argument how it was so we can maintain backwards compatibility.